readme.md

```diff
@@ -336,10 +341,27 @@ use-contrackit uses Tailwind for styling, to use the modal in dark mode simply a
 
 ## Development
 
-To run use-contractkit locally, simply clone this repository and run:
+To develop locally and see your changes reflected immediately, you can run `use-contractkit` and the `example` app in parallel.
 
-- `yarn`
-- `yarn dev`
+The steps for that are:
+
+1. Clone the repo
+
+2. Run `yarn` on the root of the project
+
+3. Run `use-contractkit`:
+
+```sh
+cd packages/use-contractkit
+yarn dev
+```
+
+4. Lastly, run the `example` app:
+
+```sh
+cd packages/example
+yarn dev
```

Comment:
if you run `yarn dev` from root it will run both at same time so so you dont need to run yarn dev in 2 browsers

Reply:
When I run yarn dev from root, it tries to run wallet connect and it fails. :(

Reply:
🤔 Now if I run it, it works, but when I cloned it initially and followed the steps it didn't work. I'll check what's missing.

Reply:
FYI, these are the errors:
```
example: wait  - compiling...
example: error - ./pages/_app.tsx:1:0
example: Module not found: Can't resolve '@celo-tools/use-contractkit/lib/styles.css'
example: > 1 | import '@celo-tools/use-contractkit/lib/styles.css';
example:   2 | import '../styles/global.css';
example:   3 |
example:   4 | import { Alfajores, ContractKitProvider } from '@celo-tools/use-contractkit';
example: https://nextjs.org/docs/messages/module-not-found
@celo-tools/use-contractkit: src/connectors/connectors.ts(17,8): error TS2307: Cannot find module '@celo/wallet-walletconnect-v1' or its corresponding type declarati
ons.
@celo-tools/use-contractkit: src/connectors/connectors.ts(420,34): error TS7006: Parameter 'error' implicitly has an 'any' type.
@celo-tools/use-contractkit: src/connectors/connectors.ts(420,41): error TS7006: Parameter 'session' implicitly has an 'any' type.
@celo-tools/use-contractkit: src/connectors/useWalletConnectConnector.ts(3,26): error TS2307: Cannot find module '@celo/wallet-walletconnect-v1' or its corresponding type declarations.
@celo-tools/use-contractkit: error Command failed with exit code 1.
@celo-tools/use-contractkit: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
@celo-tools/use-contractkit: error Command failed with exit code 1.
@celo-tools/use-contractkit: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run dev exited 1 in '@celo-tools/use-contractkit'
lerna WARN complete Waiting for 3 child processes to exit. CTRL-C to exit immediately.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
use-contractkit on  master via  v14.19.1 took 6s
```

Reply:
Running `yarn build` between `yarn` and `yarn dev` works. So I put back the original instructions but left the new text as an alternative. Let me know what you think.